src/elements.js

```diff
@@ -103,12 +103,12 @@ function getOrCreateElement(elements, index, type, initProperties) {
   return element;
 }
 
-function resolveAnnotationOptions(resolver) {
+function resolveAnnotationOptions(resolver, mainResolver) {
   const elementClass = annotationTypes[resolveType(resolver.type)];
   const result = {};
   result.id = resolver.id;
   result.type = resolver.type;
-  result.drawTime = resolver.drawTime;
+  result.drawTime = resolver.drawTime || (mainResolver && mainResolver.drawTime);
```

Comment:
This should be possible to handle with fallback

Reply:
well, I have tested several times and maybe there is another way to do it but the "resolver" is based on label annotation and the drawTime is not falling back to line annotation options. Without that, the drawTime is null (where is missing), value set in annotation options.

Reply:
Ok, I'll do some testing. Maybe this would need #630 to work properly.

Reply:
It seems when rebased to #630 the mainResolver is no longer needed.

Reply:
solved by 630